sylt-parser/src/statement.rs

```diff
@@ -466,10 +466,9 @@ pub fn statement<'t>(ctx: Context<'t>) -> ParseResult<'t, Statement> {
         // `ret <expression>`
```

Comment:
```suggestion
        // `ret [<expression>]`
```

Reply:
The syntax has changed, the expression is optional now - so that's what the comment is signifying. Is it unclear?